### PR TITLE
Fix page management

### PR DESCRIPTION
**Pull request by fkroockmann**
https://github.com/backbee/backbee-standard/issues/421

### CODE REVIEW

File: src/tb/apps/page/controllers/main.controller.js [hunk 1]
```diff
@@ -184,6 +184,7 @@ define(
                     this.manageView = new ManageView({'pageStore': PageStore});
                     this.manageView.render();
                 } else {
+                    console.log(this.manageView.popin);
```
Comment (mickaelandrieu) [inline src/tb/apps/page/controllers/main.controller.js]: remove this console.log statment pls

File: src/tb/component/popin/main.js [hunk 2]
```diff
@@ -246,6 +246,8 @@ define('tb.component/popin/main', ['Core', 'tb.component/popin/PopIn', 'jquery',
                     });
 
                     jQuery('#' + popIn.getId()).dialog('open');
```
Comment (mickaelandrieu) [inline src/tb/component/popin/main.js]: you don't need else statment, move this statment outside of if/else :)

**Comment by mickaelandrieu** [comment]
:+1 : you are such amazing, damn fuck'* good developper !

**Comment by fkroockmann** [comment]
Thanks for review @mickaelandrieu :) xD

**Pull request merged**